lib/watir/rspec/cli.rb

```diff
@@ -88,6 +88,8 @@ def install
 end
   ]
           end
+
+          puts "WARNING : the HTML formatter cannot be used until the rspec legacy formatter gem is also installed !! (https://github.com/rspec/rspec-legacy_formatters)"
```

Comment:
that is indeed an option.

however I've noticed that even with the legacy gem there is still an error when a test fails:

NoMethodError: undefined method `before_all_ivars' for RSpec::ExampleGroups::GatekeeperEvalLicenses:Class
    occurred at /rspec-core-3.2.0/lib/rspec/core/example_group.rb:634:in `method_missing'


The before_all_ivars method was indeed removed in rspec-core ;( So I guess the best solution is to rewrite the HTML formatter using the new syntax